src/tritongrader/autograder.py

```diff
@@ -175,7 +178,7 @@ def copy2sandbox(self, src_dir, item):
             shutil.copy2(path, dst)
             logger.info(f"Copied file from {path} to {dst}...")
         elif os.path.isdir(path):
-            shutil.copytree(path, dst)
+            shutil.copytree(path, dst, dirs_exist_ok=True)
```

Comment:
what's the reasoning behind this change?

Reply:
don't we want to crash if something malfunctioned (and made this target dir already)?